blog/rome-incentives/README.md

```diff
@@ -0,0 +1,237 @@
+
+The `Content Curators` (see below) will be incentivized to find content with incorrect metadata and will likely remove it from the content directory. In these cases, content creators will not be paid. Multiple spam uploads which represent a burden to moderate for the `Content Curators` may even be penalized and result in deductions on payouts due for qualifying content uploads on your content creator profile.
+
+Uploading illegal or copyrighted content will result in a disqualification from payouts. It will also result in the takedown of content, potentially slashing of funds, and the deletion of your channel.
+
+We will provide `Content Creators` with a list of sources for safe and non-copyrighted material which can be uploaded as part of this testnet and will qualify for the payments discussed above and laid out in the table below.
+
+### Rewards based on content type
+
+| CONTENT TYPE | MAXIMUM PAYMENT PER ITEM | MAXIMUM WEEKLY POOL |
+| --- | --- | --- |
+| Short Video (Standard) | $0.35 per video* | $70 |
+| Long Video (Standard) | $0.50 per video* | $100 |
+| Best Content (Bonus) | $1 per item* | $25 |
+| Original Content (Bonus) | $1 per item* | $25 |
+
+*It's important to note that the rewards listed in the table are maximums. Short videos are under four minutes and long videos exceed four minutes in length.
+
+For example, if more than 200 `short videos` are uploaded in a week the reward per
+short video item will be reduced for that week. In the case of 400 `short videos` being
+uploaded, the reward will be halved to $0.175 per file ($70 ÷ 400). For 800
+`short videos` in a week the reward would be $0.0875 and so on.
+
+Basically, the payment per item is constant until the number of items in
+category multiplied by maximum payment per item exceeds the maximum weekly
+pool. We have chosen this reward structure in order to incentivize users to
+upload a variety of different content types on Rome.
+
+Payouts for this role will be made every Monday at ~11:00GMT.
+
+Content Curators
+----------------
+
+![](Content%20Curators.png)
+
+<i>Earn your share of $375 per week</i>
+
+`Content Curators` will one day be essential for ensuring that the petabytes of media items uploaded to Joystream are formatted correctly and comprehensively monitored and moderated. Our upcoming testnet allows this content monitoring to take place by giving users who are selected for the role administrative access to the Joystream content directory to make changes where necessary.
+
+We will have five slots available for `Content Curators` when Rome is launched. They will need to apply to the `Content Curator Lead` (a `sudo` role operated by [Jsgenesis](https://jsgenesis.com/)) and will have their application assessed by this lead before being added as a curator for a default term of one week.
```

Comment:
We will open with 5 slots, but this can be changed depending on demand and activity.

The group lead will initially be occupied by Jsgenesis, but as the network matures, the role will be delegated to a community member.